mw-webapp/src/logic/waysTable/columns.tsx

```diff
@@ -1,10 +1,10 @@
 import {createColumnHelper} from "@tanstack/react-table";
 import {Link} from "src/component/link/Link";
-import {renderCellValue} from "src/logic/waysTable/renderCellValue/renderCellValue";
 import {GoalPreview} from "src/model/businessModelPreview/GoalPreview";
 import {UserPreview} from "src/model/businessModelPreview/UserPreview";
 import {WayPreview} from "src/model/businessModelPreview/WayPreview";
 import {pages} from "src/router/pages";
+import {renderCellValue} from "src/utils/renderCellValue";
```

Comment:
why this function on utils?

Reply:
I think if it is used in more than columns.tsx file - it becomes a util

Reply:
It's not a util file but let's slip it - I refactor it in new PR #190